backend/src/routes/api/roadtrip/__tests__/roadtrip.test.js

```diff
@@ -20,20 +32,40 @@ beforeAll(async done => {
 
 });
 
-beforeEach(async () => {
+beforeEach(async () => {    
     const roadTripsColl = await mongoose.connection.db.createCollection('roadtrips');
 
+    const usersColl =  mongoose.connection.db.collection("users", function(err, collection){
```

Comment:
```
const usersColl =await mongoose.connection.db.createCollection('users');
```
Ideally I would like to use the above line instead of line 38, but doing so break the tests which I am struggling to find out why